File: TracNghiem.WebAPI/Infrastructure/ModelQueries/UrlQuery.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TracNghiem.WebAPI.Infrastructure.ModelQueries
{
    public class UrlQuery
    {
        public string Keyword { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }
        public UrlQuery()
        {
            Keyword = "";
            Page = 1;
            PageSize = 10;
        }
    }
}

File: TracNghiem.FE/src/app/shared/Models/UrlQuery.ts
export class UrlQuery{
  pageNumber:number = 1;
  pageSize:number = 10;
  keyword:string = "";
  constructor(page?: number, pageSize?: number, keyword?: string){
    this.pageNumber = page || 1;
    this.pageSize = pageSize || 10;
    this.keyword = keyword || '';
  }
}
